flume-ng-channels/flume-kafka-channel/src/test/java/org/apache/flume/channel/kafka/TestKafkaChannelBase.java

```diff
@@ -120,6 +122,7 @@ Context prepareDefaultContext(boolean parseAsFlume) {
     Context context = new Context();
     context.put(BOOTSTRAP_SERVERS_CONFIG, testUtil.getKafkaServerUrl());
     context.put(PARSE_AS_FLUME_EVENT, String.valueOf(parseAsFlume));
+    context.put("offsets.topic.replication.factor", "1");
```

Comment:
What does this change bring? Did we need this to get the test working?

Reply:
Removed it now.